lib/manager/bundler/readme.md

```diff
@@ -46,78 +36,86 @@ No
 
 #### Can package files have "local" links to each other that need to be resolved?
 
+No
+
 #### Is there reason why package files need to be parsed together (in serial) instead of independently?
 
----
+No
 
 #### What format/syntax is the package file in? e.g. JSON, TOML, custom?
 
----
+Ruby syntax
 
 #### How do you suggest parsing the file? Using an off-the-shelf parser, using regex, or can it be custom-parsed line by line?
 
----
+I'm thinking about to use something that can transform ruby code to abstract syntax tree
 
 #### Does the package file structure distinguish between different "types" of dependencies? e.g. production dependencies, dev dependencies, etc?
 
-No
-
----
+Yes, but not sure that it's relevant to Renovate purposes
 
 #### List all the sources/syntaxes of dependencies that can be extracted:
 
----
+Bundler can use multiple gem sources
 
 #### Describe which types of dependencies above are supported and which will be implemented in future:
 
 ## Versioning
 
 #### What versioning scheme do the package files use?
 
----
+Semantic version scheme
 
-#### Does this versioning scheme support range constraints, e.g. `^1.0.0` or `1.x`?
+#### Does this versioning scheme support range constraints, e.g. ^1.0.0 or 1.x?
 
----
+Yes
 
 #### Is this package manager used for applications, libraries, or both? If both, is there a way to tell which is which?
 
----
+Yes
 
 #### If ranges are supported, are there any cases when Renovate should pin ranges to exact versions if rangeStrategy=auto?
 
+No
```

Comment:
dev dependencies could always be pinned, and production dependencies pinned if we determine that it's an application and not library.